DiscordRPC/RichPresence.cs

```diff
@@ -920,6 +955,41 @@ internal override bool Matches(RichPresence other)
         {
             return presesnce != null;
         }
+
+        /// <summary>
+        /// Converts this RichPresence to BaseRichPresence
+        /// </summary>
+        /// <returns></returns>
+        public BaseRichPresence ToBaseRichPresence()
```

Comment:
This would be almost never used. 
BasePresence is basically just for returned data back from discord.

Reply:
gotcha removed it